ui/job-view/headerbars/SecondaryNavBar.jsx

```diff
@@ -138,10 +138,22 @@ class SecondaryNavBar extends React.PureComponent {
     setUrlParam('group_state', groupState);
   };
 
-  toggleUnclassifiedFailures = () => {
+  toggleUnclassifiedFailures = event => {
     const { filterModel } = this.props;
 
     filterModel.toggleUnclassifiedFailures();
+
+    this.indicateUnclassifiedFailuresToggle(event);
```

Comment:
I like the approach you're taking of just making the button more yellow.  That's a good idea.

However, this would be better done by setting a ``state`` value to indicate we are showing unclassified failures.  This state would need to also be set on page-load/page-reload.

Then in the ``render`` method, you can turn on or off a particular ``.CSS`` class by using a ternary operator like this: https://github.com/hopes91/treeherder/blob/e5335206db04e2f5ac610d1f11e724bd69411ae6/ui/job-view/headerbars/SecondaryNavBar.jsx#L262-L266

In the code we have here, the button starts with the mild yellow background, then toggled on, it has the bright yellow.  But when you toggle it back of, it goes to black.  It should go back to the original background color.

Reply:
So, to be clear, you don't really need a stage property to keep track of whether it's on or not.  Just check ``FilterModel._isUnclassifiedFailures`` in the ``render`` function.

Reply:
So, to be clear, the true result of filterModel.isUnclassifiedFailures should show us that we are looking at the unclassified failures?

Reply:
Yes, that's correct.